frontend/types/interest.ts

```diff
@@ -13,8 +14,8 @@ export type Interest = {
   campaignName: string;
   campaignDescription: string;
   promisedAmount: number;
-  start: Date;
-  end: Date;
+  start: Moment | Date;
```

Comment:
Do we use `Date` type anywhere?

Reply:
We do, with yup schema casting to Date at the end of validation. Our alternatives are to ignore type checking for start / end, use a moment yup schema (ugly, and requires more custom functions), or to transform the schema just before validation. 

For the last option, I prefer being explicit that the value stored can be either moment or date, so this is the soln I stuck with. Can take this offline to discuss more if you like.